packages/tracing/src/hubextensions.ts

```diff
@@ -88,7 +90,7 @@ function sample<T extends Transaction>(hub: Hub, transaction: T, samplingContext
       `[Tracing] Discarding transaction because ${
         typeof options.tracesSampler === 'function'
           ? 'tracesSampler returned 0 or false'
-          : 'tracesSampleRate is set to 0'
+          : 'a negative sampling decision was inherited or tracesSampleRate is set to 0'
```

Comment:
Is this unrelated to the rest?

Reply:
Related only insofar as doing this work made me focus on the inheritance piece and realize that the log message was missing a possibility.